Library/Homebrew/cmd/install.rb

```diff
@@ -91,6 +91,14 @@ def install
         puts_columns(search_formulae(query))
         puts "Searching taps..."
         puts_columns(search_taps(query))
+
+        if HOMEBREW_REPOSITORY.exist? &&
+           (Time.now.strftime("%s").to_i - `stat -f "%m" #{HOMEBREW_REPOSITORY}/.git/refs/heads/master`.to_i) > 172800
```

Comment:
Would be good to have the magic number (`172800`) broken down a bit more so it's more obvious how long this is.